main/main.cpp

```
@@ -9,48 +9,45 @@
+    sensor.initializeBus();
+
+/* Busy loop for initialization, because the main loop does not work without
+ * a sensor.
+ */
+while (!sensor.probe()) {
+    printf("SHT sensor probing failed\n");
+}
+printf("SHT sensor probing successful\n");
+
+// Set measurement interval and callback, then start continuous measurements
+sensor.setMeasurementInterval(5000); // 1 second interval
```

Clang (14.0.6)

[warning] 5000 is a magic number; consider replacing it with a named constant

(cppcoreguidelines-avoid-magic-numbers,readability-magic-numbers)